documentation/smg/02_development_environment.md

```diff
@@ -1 +1,102 @@
+
+This appears to be a one-user license and I am not necessarily expecting that you will be able to install on multiple machines e.g. personal and work machine. But it may work.
+
+NB I had problems trying to restart the download - it said I did not have access rights. Clearing cookies from the page solved this.
+
+#### Using MATLAB
+
+##### Prerequisites
+
+* You need to install `cmake`. I installed the current (at time of writing) version 3.18.2; there is a minimum version but all current versions should be OK for this. Select that `cmake` is on the path.
+* You need to install the Visual Studio SDK. This can be the actual SDK or a working VS installation. I installed Visual Studio 16 2019 community edition (it's free). As (currently) cloned from github, Horace will cache a reference to VS 2017 and will say that it cannot find VS. In explorer or otherwise, go to Horace/build in the Horace repo and delete or rename CMakeCache.txt before building. 
+* You need to build Horace and Herbert. With both cloned as in the instructions above, open a Powershell and cd to the main Horace repo directory. Then type
+  * `.\tools\build_config\build.ps1 -build -vs_version 2019`
+* The build will tailor the `horace_on` and `herbert_on` commands to fit the location of the current repo.
+* You can now access Horace: cd to Horace\build\local_init and type `horace_on`. To avoid this indirect access, you can set this directory to be searched by setting the environment variable MATLABPATH to the full directory path before launching MATLAB. `horace_on` can then be used immediately on launch.
+
+### IDaaS environment
+
+#### Access
+
+* Location for login and registration is https://isis.analysis.stfc. This should open on the login page - as a new user you may get more info going directly to the registration, but if you don't, the FAQ menu at the top will take you to a page with "How do I get a user office account" three quarters of the way down the left "Contents" sidebar. This section has a link to a page with sign-in and registration areas. Bottom right click on "Create a new facility user account with us".
+* On this registration page, fill in the form. Email should be your STFC email. For "Facilities You Use" select ISIS. Click "Create New Account" at the bottom. You will probably get an email from FacilitiesBusinessSystem@stfc.ac.uk asking you to validate your account - please do.
+* After this you should be able to log in and create a password. Login is your STFC email address. Password will NOT be your federated password. You already have the federated password for access with your federated user id (form: aaa11111), and a UKSBS Oracle password for access with your  Oracle user id (form aaaa11@ph.rc). Choose whatever password you think is appropriate for this additional login which is not the same as the other two. NB the federated password never seems to change but is obscure; you have to change the Oracle password every 90 days.
```

Comment:
```suggestion
* After this you should be able to log in and create a password. Login is your STFC email address. Password will NOT be your federated password. You already have the federated password for access with your federated user id (form: aaa11111), and a UKSBS Oracle password for access with your Oracle user id (form aaaa11@ph.rc). Choose whatever password you think is appropriate for this additional login which is not the same as the other two. NB the federated password never seems to change but is obscure; you have to change the Oracle password every 90 days.
```

Reply:
Sorry, I can't see a change - what am I missing?

Reply:
Double space before "Oracle user id" ?

Reply:
Ah sorry, remove double space before Oracle user ID. Anything else?

Reply:
No, that was it